exercises/palindrome-products/src/example.h

```diff
@@ -1,12 +1,27 @@
 #ifndef PALINDROME_PRODUCTS_H
 #define PALINDROME_PRODUCTS_H
 
-typedef struct {
+#define MAXERR 100
+
+struct factors {
+   int factor_a;
+   int factor_b;
+   struct factors *next;
+};
+
+typedef struct factors factor_t;
```

Comment:
No need to change, but you can do this more simply with:

```c
typedef struct factors {
   int factor_a;
   int factor_b;
   struct factors *next;
} factor_t;
```

And if you don't intend to ever declare one of these as `struct factors` (and instead plan to only use `factor_t`), you can simplify further to:

```c
typedef struct {
   int factor_a;
   int factor_b;
   struct factors *next;
} factor_t;
```